qemu_v8.mk

```diff
@@ -498,6 +514,34 @@ else
 QEMU_MTE	= off
 endif
 
+QEMU_ARGS = -nographic
+QEMU_ARGS += -smp $(QEMU_SMP)
+QEMU_ARGS += -cpu $(QEMU_CPU)
+QEMU_ARGS += -d unimp -semihosting-config enable=on,target=native
+QEMU_ARGS += -m $(QEMU_MEM)
+QEMU_ARGS += -bios bl1.bin
+QEMU_ARGS += -initrd rootfs.cpio.gz
+QEMU_ARGS += -kernel Image
+QEMU_ARGS += -append 'console=ttyAMA0,38400 keep_bootcon root=/dev/vda2 $(QEMU_KERNEL_BOOTARGS)'
```

Comment:
In fact, 525 is needed because it's added in the dtb but I agree that 522:524 are not. Just wanted minimize the number of defines.

I'm going to split with your naming proposal. I don't have better name

Reply:
But -append param mandates to have -kernel one.
And after checking the dtb generated by qemu, it is different without -bios and the kernel doesn't boot
So we must have all parameters to generate the dtb file. Addig touch on files seems to be enough. Don' t know if this is acceptable ?

```shell
$(SCMI_DTBO): $(SCMI_DTSO)
        mkdir -p $(BINARIES_PATH)
+       touch $(BINARIES_PATH)/Image
+       touch $(BINARIES_PATH)/bl1.bin
+       touch $(BINARIES_PATH)/rootfs.cpio.gz
        dtc -I dts -O dtb -o $(SCMI_DTBO) $(SCMI_DTSO)
```

Reply:
> But -append param mandates to have -kernel one. And after checking the dtb generated by qemu, it is different without -bios and the kernel doesn't boot So we must have all parameters to generate the dtb file. Addig touch on files seems to be enough. Don' t know if this is acceptable ?

Hmmm... no. Let's add the full dependencies then (`linux arm-tf buildroot` I suppose). They are needed to do anything useful anyways :wink:

Reply:
added the dependencies (linux arm-tf buildroot) and the link to the the rootfs
`ln -sf $(ROOT)/out-br/images/rootfs.cpio.gz $(BINARIES_PATH)/`